src/index.js

```diff
@@ -15,6 +14,9 @@ import { ApolloLink } from "apollo-link";
 import { InMemoryCache } from "apollo-cache-inmemory";
 import { ApolloProvider } from "react-apollo";
 
+import { Row, Col, Button, Tabs, Menu, Dropdown, Icon, Table, List, Avatar, Badge, Card, Tooltip } from 'antd';
+import './App.css';
```

Comment:
mind keeping the existing file name convention - lowercase and underscores? uppercase files can be a hassle to work with on mixed-case sensitive systems

Reply:
App.css no worries. Will see what we can do in relation to those antd components